### ISSUE TITLE
zoom-us: Installation fails with "variable $src or $srcs should point to the source"

### ISSUE THREAD

**Issue by alexeymuranov** [NixOS/nixpkgs#120226]
**Describe the bug**

On my NixOS 20.09, i decided to install `zoom-us` from `nixpkgs-unstable`, then I removed it, and tried to reinstall from `nixos-20.09`. However, now i cannot install it from either channel, I get:
```console
$ nix-env -iA nixos.zoom-us                                                                                                                                                       ~/
installing 'zoom-5.6.16775.0418'
these derivations will be built:
  /nix/store/3ryxkdkw12nsdh5wlph9lzqfmj2xyny2-zoom-5.6.16775.0418.drv
building '/nix/store/3ryxkdkw12nsdh5wlph9lzqfmj2xyny2-zoom-5.6.16775.0418.drv'...
unpacking sources
variable $src or $srcs should point to the source
builder for '/nix/store/3ryxkdkw12nsdh5wlph9lzqfmj2xyny2-zoom-5.6.16775.0418.drv' failed with exit code 1
error: build of '/nix/store/3ryxkdkw12nsdh5wlph9lzqfmj2xyny2-zoom-5.6.16775.0418.drv' failed
```

I tried collecting garbage, rebooting, and I also did
```console
$ sudo nix-store --verify --check-contents --repair
```
but nothing helps.

**To Reproduce**
1. For me, this fails:
   ```console
   $ nix-env -iA nixos.zoom-us                                                                                                                                                       ~/
   ```

**Expected behavior**
Should install Zoom.

**Additional context**
Latest Zoom from stable channel was working until I first installed the version from  `nixpkgs-unstable` and then removed it.

**Notify maintainers**
@danbst @tadfisher @doronbehar

**Metadata**
```console
$ nix-shell -p nix-info --run "nix-info -m"                                                                                                                                       ~/
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.111-hardened1, NixOS, 20.09.3888.4263ba5e133 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.10`
 - channels(alexey): `"unstable-21.05pre282987.294d1925af6"`
 - channels(root): `"nixos-20.09.3888.4263ba5e133"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  zoom-us
# a list of nixos modules affected by the problem
module:
```

**Comment by wamserma** [comment]
This was broken with one of the recent channel updates:

```
working: "20.09.3882.5c0e6a8c319"
broken:   "20.09.3888.4263ba5e133"
```

**Comment by doronbehar** [comment]
Sorry about that. Fix should work from #120242 .

**Comment by alexeymuranov** [comment]
Fixed, thanks.
